vpr/src/draw/draw.cpp

```diff
@@ -154,13 +158,38 @@ static void set_block_text(GtkWidget* widget, gint /*response_id*/, gpointer /*d
 static void clip_routing_util(GtkWidget* widget, gint /*response_id*/, gpointer /*data*/);
 static void run_graphics_commands(std::string commands);
 
```

Comment:
draw.cpp is pretty big.  Could these functions be moved to another file (e.g. draw_debug.cpp)?

Reply:
resolved